tests/integration/features/command/active-calls.feature

```diff
@@ -0,0 +1,41 @@
+Feature: create
+
+  Background:
+    Given user "participant1" exists
+
+  Scenario: No call in progress
+    Given invoking occ with "talk:active-calls"
+    Then the command was successful
+    And the command output contains the text "No calls in progress"
+
+  Scenario: Users only chatting
+    When user "participant1" creates room "room"
+      | roomType | 3 |
+      | roomName | room |
+    Then user "participant1" joins room "room" with 200
+
+    Given invoking occ with "talk:active-calls"
+    Then the command was successful
+    And the command output contains the text "No calls in progress"
+
+    Then user "participant1" leaves room "room" with 200
+
+
+  Scenario: Call in progress
+    When user "participant1" creates room "room"
+      | roomType | 3 |
+      | roomName | room |
+
+    Then user "participant1" joins room "room" with 200
+    And user "participant1" joins call "room" with 200
+
+    Given invoking occ with "talk:active-calls"
+    Then the command failed with exit code 1
```

Comment:
why would the command fail but still reveal proper information ?

if legit, add a comment above this line with the answer

Reply:
it `return 1`s so you can loop it in a bash script to alert you when all calls finally ended and it's save to e.g. update.
But nothing new in this PR, is there since quite a bit only the test is new.

Reply:
https://github.com/nextcloud/spreed/pull/4496/commits/66c8bc5b0d263144068bfde6162fc81b660da9cf